run_tests.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Run tests for XBlock-utils
```

Comment:
The more I think about it the more I'd like to see this - you've got a lot of tests here and it's hard to tell if they're passing or not - relying on running them locally isn't a good idea. If you open up this PR and tag me, we can get it reviewed & merged quickly.